Assets/HoloToolkit/Input/Prefabs/InputManager.prefab

```diff
@@ -81,6 +81,22 @@ GameObject:
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
   m_IsActive: 1
+--- !u!1 &1547260285441492
+GameObject:
+  m_ObjectHideFlags: 0
+  m_PrefabParentObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 100100000}
+  serializedVersion: 5
+  m_Component:
+  - component: {fileID: 4277215840702346}
+  - component: {fileID: 114080562701649166}
+  m_Layer: 0
+  m_Name: TouchInputControl
```

Comment:
Is this something we want by default on the InputManager prefab?

Does it / can it remove itself when run on devices without a touch screen?

Reply:
Hi @keveleigh  - this was requested/added on Nov 29 (see earlier in this thread). I hadn't initially intended it to be present by default but other contributions have added other device handlers such as mouse and xbox controller, though to be fair I think they're event based rather than polling.
Happy to add a check for touchSupported at Start() and self-disable, if that would satisfy?

Reply:
Ohh interesting, I must have missed that!

Yeah, a check for touchSupported would be perfect, in my opinion

Reply:
If this is the case, should we also add `TouchSupported` in our SupportedInputInfo enum?

Reply:
`SupportedInputInfo` tells you the features of an input source that you can ask it about. From my understanding of this feature, there aren't any touch-specific features to query, as all events are routed through existing ones.

I see this like the Xbox controller script which routes tap/hold/navigation events through the system, where we didn't add "Xbox controller" to that enum.

Reply:
Good point. Maybe we should have?

Reply:
I don't think so.

I see "Xbox controller" and "touch screen" in the same category as "motion controller", which are input source types.

I see the SupportedInputInfo enum telling you about features you can query on the source itself, like the state of the menu button or the position of the input. As-is, this new source doesn't add any new features that a source can have, but it does add a new type of source that can be used.